tests/test_poll.py

```diff
@@ -15,4 +16,4 @@ def test_simple_api(cluster, poll, monkeypatch):
             map_client.set('key:%s' % x, x)
 
     for x in range(10):
-        assert client.get('key:%d' % x) == str(x)
+        assert client.get('key:%d' % x) == bytes_type(x)
```

Comment:
Probably same here too

Reply:
I think you're probably right, lets see what happens in py3 land.

Going to leave this as is for now since tests pass

Reply:
I think we should fix this now, we're passing ints to `bytes` in python 3, which will generate a giant bytestring and fail. `text_type` will work fine here.

Reply:
oh doy, you're right